ui/app/src/components/MarketplaceDialog/MarketplaceDialog.tsx

```diff
@@ -0,0 +1,223 @@
+/*
```

Comment:
This name might be a little vague. Perhaps something like "InstallPluginDialog"?